app/scripts/directives/seekBar.js

```diff
@@ -30,6 +30,10 @@
                     return {width: percentString()};
                 };
                 
+                scope.thumbStyle = function(){
+                    return {width: 10};
```

Comment:
how do you make the percentage bar more?  Should left always be 10.. or will it need to change?